cabal.project

```diff
@@ -1,5 +1,19 @@
--- run `nix flake lock --update-input hackageNix` after updating index-state.
+-- Internal repository for cardano haskell packages, see CONTRIBUTING for more
```

Comment:
```suggestion
-- Custom repository for cardano haskell packages, see CONTRIBUTING for more
```

Reply:
Foliage describe itself as a tool to create custom or private Haskell package repositories. Suggest to use the word 'custom' instead of 'internal' per erikd remarks as the repo is indeed public (https://input-output-hk.github.io/cardano-haskell-package-repo/)